File: frontend/src/models/NormalPorConteo.dto.ts
export interface NormalPorConteoDto {
  id?: number | null;
  activo?: boolean;
  germinacionId?: number | null;
  tabla?: 'SIN_CURAR' | 'CURADA_PLANTA' | 'CURADA_LABORATORIO' | string;
  numeroRepeticion?: number | null;
  conteoId?: number | null;
  normal?: number | null;
}

File: src/main/java/ti/proyectoinia/dtos/NormalPorConteoDto.java
package ti.proyectoinia.dtos;

import lombok.Data;

@Data
public class NormalPorConteoDto {
    private Long id;
    private boolean activo;

    private Long germinacionId;
    private String tabla; // SIN_CURAR | CURADA_PLANTA | CURADA_LABORATORIO
    private Integer numeroRepeticion;
    private Long conteoId;

    private Integer normal;
    private Float promedioNormal;
}
